docs/source/changelog.rst

```diff
@@ -4,6 +4,94 @@
 Changes in nbconvert
 ====================
 
+5.6
+---
+
+The following 24 authors and reviewers contributed 224 commits -- Thank you all!
+* 00Kai0
+* Aidan Feldman
+* Alex Rudy
+* Alexander Kapshuna
+* Alexander Rudy
+* amniskin
+* Carol Willing
+* Dustin H
+* Hsiaoming Yang
+* imtsuki
+* Jessica B. Hamrick
+* KrokodileDandy
+* Kunal Marwaha
+* Matthew Seal
+* Matthias Geier
+* Miro Hrončok
+* M Pacer
+* Nils Japke
+* njapke
+* Sebastian Führ
+* Sylvain Corlay
+* Tyler Makaro
+* Valery M
+* Wayne Witzel
+
+The full list of changes they made can be seen `on GitHub <<https://github.com/jupyter/nbconvert/issues?q=milestone%3A5.6+>`__
+
+Significant Changes
+~~~~~~~~~~~~~~~~~~~
+
+Parallel NBConvert
+++++++++++++++++++
+NBConvert --execute can now be run in parallel via threads, multiprocessing, or async patterns! This means you can now parallelize nbconvert via a bash loop, or a python concurrency pattern and it should be able to execute those notebooks in parallel.
+
+Not all kernels are safe to execute concurrently. The ipython kernel as of version 1.5.2, for python 3, should now be safe to run concurrently. But the python 2 version for ipython sometimes fails with a socket bind exception. Other kernels were not tested heavily.
```

Comment:
Kernels have varying support for safe concurrent execution. The ipython kernel (``ipykernel`` version 1.5.2 and higher) should be safe to run concurrently using Python 3. However, the Python 2 ``ipykernel`` does not always provide safe concurrent execution and sometimes fails with a socket bind exception. Unlike ``ipykernel`` which is maintained by the project, other community-maintained kernels may have varying support for concurrent execution, and these kernels were not tested heavily.